src/components/Uploader/Uploader.stories.tsx

```diff
@@ -0,0 +1,39 @@
+/** @jsxRuntime classic /*
+/** @jsx jsx */
+import { jsx, Text } from "theme-ui"
+
+import React, { useState } from "react"
+import { Meta, Story } from "@storybook/react/types-6-0"
+
+import Uploader, { IUploaderProps } from "./Uploader"
+
+export default {
+  title: "Design System/Uploader",
+  component: Uploader,
+} as Meta
+
+export const DefaultTemplate: Story<IUploaderProps> = (props) => {
```

Comment:
The template shouldn't be exported, it's being rendered as a story
```suggestion
const DefaultTemplate: Story<IUploaderProps> = (props) => {
```